src/series.rs

```diff
@@ -821,6 +821,22 @@ impl JsSeries {
         Ok(s.into())
     }
 
+    #[napi(catch_unwind)]
+    pub fn str_json_extract(
+        &self,
+        dtype: Option<Wrap<DataType>>,
+        infer_schema_len: Option<i64>,
+    ) -> napi::Result<JsSeries> {
+        let ca = self.series.utf8().map_err(JsPolarsErr::from)?;
+        let dt = dtype.clone().map(|d| d.0);
```

Comment:
i don't think we need the clone. 
```suggestion
        let dt = dtype.map(|d| d.0);
```